build-system/tasks/check-links.js

```diff
@@ -166,13 +166,18 @@ function filterWhitelistedLinks(markdown) {
  * @param {string} markdownFile Path of markdown file, relative to src root.
  * @return {Promise} Used to wait until the async link checker is done.
  */
-function runLinkChecker(markdownFile) {
+async function runLinkChecker(markdownFile) {
+  // The extension generator markdown template has placeholder links that can't
+  // be checked.
+  if (/doc-template\.md$/.test(markdownFile)) {
```

Comment:
nit: possibly better as a constant and probably better with the case-insensitive flag and and `(^|/)`; alternately, `path.basename(markdownFile).toLowerCase() === 'doc-template.md'`

Reply:
I'll set `-TEMPLATE` (case sensitive) as a common suffix for markdown files to be ignored by tooling checks.